configs/config_map.json

```diff
@@ -0,0 +1,10 @@
+{
+    "IBMCompatibleSystems": {
```

Comment:
I think you could probably change this to something  that isn't IBM specific, since it just maps device tree model names to subdir names.

Reply:
I kept it having following thoughts in mind:
a)  The Interface which is added under EM has the same name. ("xyz.openbmc_project.Configuration.IBMCompatibleSystem")

b)  And as these system names are IBM specific and led config folders already have "ibm" in their name it will be more synchronous to have such name.

Please let me know if you have some other thoughts here.

Reply:
If you're not going to put this upstream, then I'm not  too concerned.

If you are going to try to upstream it, then we probably wouldn't want to hard code IBM here again, because nothing is stopping someone else from wanting to use this feature and putting more led-group-config.jsons in subdirectories under the configs directory.

Depending on who the reviewers are, they might also suggest just making the subdirectory just match with the model name exactly.

Reply:
I would also prefer to get the IBMCompatibleSystem to a generic name , so that anyone can add on top of these for their specific system.
further I would keep the name as <company>,<model name> as multiple companies could have the same model name. There is no restriction in that.

Reply:
Changes Done.